src/responder/kcm/kcmsrv_ccache_secdb.c

```diff
@@ -79,73 +71,35 @@ static errno_t sec_get_b64(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
-static errno_t sec_put_b64(TALLOC_CTX *mem_ctx,
-                           struct sss_sec_req *req,
-                           struct sss_iobuf *buf)
+static errno_t sec_put(TALLOC_CTX *mem_ctx,
```

Comment:
So this function now does two things: call `sss_sec_put()`, and log on failure.  It has two callsites - one of which logs failure again, and the other which doesn't.  It seems like that should be adjusted - my inclination would be to add logging to the other callsite and eliminate this function entirely.  However, with this change, `sec_update()` ends up in a similar place except that none of its callsites log, so the smaller change is probably to eliminate the extra log (in `ccdb_secdb_create_send()`).

Reply:
Yeah, these function got smaller and smaller until they do nothing else then call to sss_secrets. But I prefer to keep them to be consistent with sss_get. I removed the log message.